auth/docs/adding-icons.md

```diff
@@ -7,6 +7,15 @@ If you would like to add your own custom icon, please open a pull-request with
 the relevant SVG placed within `assets/custom-icons/icons` and add the
 corresponding entry within `assets/custom-icons/_data/custom-icons.json`.
 
+Please be careful to upload small and optimized icon files. If your icon file 
+is over 50KB, it is certainly not optimized.
```

Comment:
Can we change "certainly" to "likely" so that it comes off as a bit more friendlier? Otherwise LGTM!

Reply:
Definitely a good observation, please go ahead.